### PR TITLE
Create website-maintenance splash page for old-weather-whaling.

### PR DESCRIPTION
**Pull request by jelliotartz**
A couple of issues:

1. Karla font - currently imported from google fonts via link tag in head. Could these be imported more efficiently?

2. Links to alternate site and discussion page - are these urls correct?

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  background-color: #efefef;
+}
+
+.container {
+  position: absolute;
+  width: 50%;
+  top: 30%;
+  left: 25%;
+  text-align: center;
+  font-family: Karla;
```
Comment (srallen) [inline main.css]: declare the font targeting `html` instead. You can declare the base size, weight, and line height for the font as well as the fallback: https://developer.mozilla.org/en-US/docs/Web/CSS/font?v=example

In this case, something like this: `font: 400 16px/1.5 "Karla", sans-serif`

File: index.html [hunk 2]
```diff
@@ -1,32 +1,22 @@
...
-    <p>Please check back soon.</p>
-
-    <p id="logo"><a href="https://www.zooniverse.org/"><img src="logo.png" alt="Zooniverse"></a></p>
-</div>
-</body>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
+  </head>
+  <body>
+    <img src="img/shape.svg">
```
Comment (srallen) [inline index.html]: Set this as a background image in the CSS instead. Check out the `background-image` and `background-repeat` properties.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #efefef;
+}
+
+.container {
+  position: absolute;
```
Comment (srallen) [inline main.css]: Once the SVG logo is set as the background, you won't need to absolutely position this.

Comment (srallen) [reply main.css]: Check out the flex spec for centering content. It's super easy when you get the hang of it: https://www.kirupa.com/html5/centering_vertically_horizontally.htm

File: main.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  font-size: 45px;
+  line-height: 1.16;
+  color: #5c5c5c;
+}
+
+.old-weather-whaling-text {
+  font-size: 14px;
+}
+
+.link-text {
```
Comment (srallen) [inline main.css:42]: Links should have an indication of state change on hover and on focus for accessibility. Easiest is to make the color slightly lighten and you can target these using the `:hover` and `:focus` pseudo-classes.

File: index.html [hunk 5]
```diff
@@ -1,32 +1,22 @@
...
+    <div class="container">
+      <a href="https://www.zooniverse.org/">
+        <img src="img/zooniverse-logo.svg" class="zooniverse-logo">
+      </a>
+      <div class="title">OLD WEATHER WHALING</div>
+      <h1 class="maintenance-msg">Down for maintenance</h1>
+      <p class="old-weather-whaling-text">
+        Old Weather Whaling is currently down for maintenance.
<br>
```
Comment (srallen) [inline index.html]: Something wrong with the period at the end of the sentence. Delete it and make sure it's just text?

File: index.html [hunk 6]
```diff
@@ -1,32 +1,22 @@
...
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
+  </head>
+  <body>
+    <img src="img/shape.svg">
+    <div class="container">
+      <a href="https://www.zooniverse.org/">
+        <img src="img/zooniverse-logo.svg" class="zooniverse-logo">
+      </a>
+      <div class="title">OLD WEATHER WHALING</div>
```
Comment (srallen) [inline index.html]: You can make caps using CSS using `text-transform: uppercase`

**Review by srallen — CHANGES_REQUESTED** [review]
Re the questions: using the link tag is fine. @trouille should be able to answer which project should be linked to.

**Comment by trouille** [comment]
@jelliotartz The Discussion Forum link should point to https://www.zooniverse.org/projects/zooniverse/old-weather/talk 

(it's the only way people will be able to access that discussion forum while the site is down. I don't expect much traffic, if any there, but good to give access). 

The other link (to https://www.oldweather.org/) is correct. From there, people can navigate to http://forum.oldweather.org/, so we don't need to link to that discussion forum in the maintenance page. 

Thanks!

File: main.css [hunk 7]
```diff
@@ -2,17 +2,17 @@ body {
...
+  height: 100vh;
   display: flex;
   justify-content: center;
-  align-self: center;
+  align-items: center;
 }
 
 .content {
   text-align: center;
   font: 400 16px/1.5 Karla, sans-serif;
```
Comment (srallen) [inline main.css]: This should be applied to `html` rather than a class.

File: main.css [hunk 8]
```diff
@@ -2,17 +2,17 @@ body {
...
+  margin: 3vw;
```
Comment (srallen) [inline main.css]: `vw` is a width unit and this is setting it all around. How about: `margin: 1em 3vw`?

**Comment by srallen** [comment]
This looks ready for use, but as we discussed, the branch won't be merging into master and we'll keep this as a styled project specific version.

**Comment by adammcmaster** [comment]
Could we maybe put the project specific versions in folders and keep them all in master? (Similar to the [retirement pages](https://github.com/zooniverse/retirement))

**Pull request closed without merging**